examples/client/Locomotion/src/services/network.js

```diff
@@ -68,9 +68,14 @@ class Network {
           AppSettings.getOperationId(),
         ]);
         this.axios.defaults.baseURL = baseURL;
-        const accessToken = await Auth.getAT(this.axios);
+        const [accessToken, captchaToken] = await Promise.all([
+          Auth.getAT(this.axios),
+          Config.CAPTCHA_KEY && Auth.getCaptchaToken(),
+        ]);
+        if (!accessToken && captchaToken) {
+          this.axios.defaults.headers.common['x-captcha-token'] = captchaToken;
+        }
```

Comment:
you need to check only if captchaToken exists (it doesnt matter if there is accessToken or not)
and I think that once you insert token to the local storage, it stays forever
im don't see where you delete it